.github/instructions/stack-updates.instructions.md

```diff
@@ -0,0 +1,264 @@
+# Stack Updates Workflow Instructions for Oryx
+
+This document provides comprehensive instructions for AI assistants working with the Oryx repository to handle stack version updates. This applies to all supported stacks: dotnet, node (nodejs), python, php
+
+## Overview
+
+The Oryx stack update workflow involves two main components:
+1. **SDK Publishing**: Building and publishing SDK artifacts to Azure storage accounts
+2. **Runtime Image Building**: Creating runtime images that reference the published SDKs
+
+## Key Files and Structure
+
+### 1. Central Configuration - `images/constants.yml`
+- **Purpose**: Central version configuration for runtime image building
+- **Content**: Version numbers, SHA256 hashes, GPG keys, and OS flavor mappings
+- **Format**:
+  ```yaml
+  node18Version: 18.20.8
+  node20Version: 20.19.3
+  node22Version: 22.17.0
+  php81Version: 8.1.33
+  php81Version_SHA: 9db83bf4590375562bc1a10b353cccbcf9fcfc56c58b7c8fb814e6865bb928d1
+  python39Version: 3.9.23
```

Comment:
should we add example for GPG key too? (python)
also ASPNETCORE vs NETCORE SHA
I think more examples here would help